librecval/extract_phrases.py

```diff
@@ -602,5 +631,11 @@ def get_mic_id(name: str) -> int:
         re.VERBOSE,
     )
     if not m:
+        if name in KNOWN_BAD_ANNOTATION_FILES:
+            raise InvalidAnnotationError(
```

Comment:
Is this going to stop the annotation process? Or just move on peacefully?

Reply:
This can and **should** stop the import process, because the annotations files identified here should **NOT** be imported—they are incomplete or incorrect annotation files :/